can/interfaces/socketcan/__init__.py

```diff
@@ -5,6 +5,5 @@
 See: https://www.kernel.org/doc/Documentation/networking/can.txt
 """
 
-from can.interfaces.socketcan import socketcan_constants as constants
-from can.interfaces.socketcan.socketcan_ctypes import SocketcanCtypes_Bus
-from can.interfaces.socketcan.socketcan_native import SocketcanNative_Bus
+from can.interfaces.socketcan import constants
```

Comment:
Should these really be "public"?

Reply:
Hm. But this really is just an implementation detail that nobody should care about. I doubt we would touch a lot of code with that change.

Reply:
Sure, I'll remove it. Since the module is now named "constants" it will work anyway.